test/gtest/gmock-gtest-all.cc

```diff
@@ -13983,46 +13984,50 @@ MockObjectRegistry g_mock_object_registry;
 
 // Maps a mock object to the reaction Google Mock should have when an
 // uninteresting method is called.  Protected by g_gmock_mutex.
-std::map<const void*, internal::CallReaction> g_uninteresting_call_reaction;
+std::unordered_map<uintptr_t, internal::CallReaction>&
+UninterestingCallReactionMap() {
+  static auto* map = new std::unordered_map<uintptr_t, internal::CallReaction>;
+  return *map;
+}
```

Comment:
Is this a change from mainstream gmock?

Reply:
NVM, I see from the commit history that it is.

Reply:
Yes:

https://github.com/google/googletest/commit/0320f517fd920866d918e564105d68fd4362040a#diff-0dfa2d04b973dc072009e740a0c47e0524bd1650c1336470f70f3437c15a9488R540

I would suggest replacing it with:

```c++
std::unordered_map<uintptr_t, internal::CallReaction>&
UninterestingCallReactionMap() {
  static std::unordered_map<uintptr_t, internal::CallReaction> umap;
  return umap;
}
```